### PR TITLE
CI and helm stuff

### PR DESCRIPTION
**Pull request by r7vme**
Add CI and Helm stuff. Plus some changes required for out CI.

### CODE REVIEW

File: helm/calico-node-controller-chart/templates/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: calico-node-controller
+  namespace: kube-system
+data:
+  etcd_endpoints: {{ .Values.Installation.V1.Infra.CalicoNodeController.EtcdEndpoint }}
+  etcd_ca: {{ .Values.Installation.V1.Infra.CalicoNodeController.EtcdCaCert }}
```
Comment (xh3b4sd) [inline helm/calico-node-controller-chart/templates/configmap.yaml]: Nitpicking, the wording I am aware of so far was `crt`, not `cert`. Would like to keep it that way. Also to keep the naming consistent, it should be `etcd_ca` and `EtcdCA`.

File: controller.go [hunk 2]
```diff
@@ -24,14 +26,41 @@ import (
...
+	// Print version.
+	if (len(os.Args) > 1) && (os.Args[1] == "version") {
+		fmt.Printf("Description:    %s\n", description)
+		fmt.Printf("Git Commit:     %s\n", gitCommit)
+		fmt.Printf("Go Version:     %s\n", runtime.Version())
+		fmt.Printf("Name:           %s\n", name)
+		fmt.Printf("OS / Arch:      %s / %s\n", runtime.GOOS, runtime.GOARCH)
+		fmt.Printf("Source:         %s\n", source)
+		return
+	}
```
Comment (xh3b4sd) [inline controller.go]: Sweet.

File: helm/calico-node-controller-chart/templates/serviceaccount.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: calico-node-controller
+  namespace: kube-system
```
Comment (xh3b4sd) [inline helm/calico-node-controller-chart/templates/serviceaccount.yaml]: For my interest, why is there a service account manifest? I never saw it for our other services.

Comment (calvix) [reply helm/calico-node-controller-chart/templates/serviceaccount.yaml]: RBAC would be my guess

Comment (r7vme) [reply helm/calico-node-controller-chart/templates/serviceaccount.yaml]: exactly, at the moment it's not used. As we do not use RBAC in host cluster.

**Comment by r7vme** [comment]
@xh3b4sd addressed your comments regarding naming [here](https://github.com/giantswarm/calico-node-controller/pull/5/commits/39c1b3e763146cd740351a17fc54cb5b59683520). 

Reflected changes also [here](https://github.com/giantswarm/installations/pull/117)

**Pull request merged**